### PR TITLE
Add ChangeLog entry regarding debug info in system libraries. NFC.

### PR DESCRIPTION
**Pull request by sbc100**
See #13078

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
 
 Current Trunk
 -------------
+- System libraries are now compiled with debug info (`-g`).  This doesn't not
```
Comment (kripken) [inline ChangeLog.md:23]:
```suggestion
- System libraries are now compiled with debug info (`-g`).  This doesn't
```

File: ChangeLog.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ See docs/process.md for more on how version tagging works.
...
+  effect release builds (builds without `-g`) but allows DWARF debugging of
```
Comment (kripken) [inline ChangeLog.md:24]:
```suggestion
  affect release builds (builds without `-g`) but allows DWARF debugging of
```

**Review by kripken — APPROVED** [review]
Thanks!

**Comment by sbc100** [comment]
See #13078

**Pull request merged**